project/config/niaomi/config/facets.facet.news_type.yml

```diff
@@ -0,0 +1,73 @@
+uuid: 84a82f60-19a9-4bdb-8f4c-fee8f0a1e3af
+langcode: en
+status: true
+dependencies:
+  config:
+    - search_api.index.news
+    - views.view.news_search
+  module:
+    - search_api
+id: news_type
+name: 'News type'
+weight: 0
+min_count: 1
+missing: false
+missing_label: others
+url_alias: type
+facet_source_id: 'search_api:views_page__news_search__news_search_page'
+field_identifier: field_news_type
+query_operator: and
+hard_limit: 0
+exclude: false
+use_hierarchy: false
+keep_hierarchy_parents_active: false
+hierarchy:
+  type: taxonomy
+  config: {  }
+expand_hierarchy: false
+enable_parent_when_child_gets_disabled: true
+widget:
+  type: links
+  config:
+    show_numbers: true
+    soft_limit: 20
+    show_reset_link: false
+    reset_text: 'Show all'
+    hide_reset_when_no_selection: false
+    soft_limit_settings:
+      show_less_label: 'Show less'
```

Comment:
I'm just being picky, but I think this should be 'Show fewer' like the other facet